ytsync/manager.go

```diff
@@ -144,12 +159,20 @@ func (s SyncManager) MarkVideoStatus(channelID string, videoID string, status st
 		vals.Add("claim_name", claimName)
 	}
 	if failureReason != "" {
+		maxReasonLength := 500
+		if len(failureReason) > maxReasonLength {
+			failureReason = failureReason[:500]
```

Comment:
DRY: `failureReason = failureReason[:maxReasonLength]`

Reply:
woops, this is fixed downstream, i have to rebase to get it here. thanks